vtkVmtk/vmtkFunctions.cmake

```diff
@@ -66,7 +66,7 @@ function(vmtk_build_library)
       KIT_PYTHON_LIBRARIES ${vmtk_wrapped_libs}
       KIT_INSTALL_BIN_DIR ${VTK_VMTK_INSTALL_BIN_DIR}
       KIT_INSTALL_LIB_DIR ${VTK_VMTK_INSTALL_LIB_DIR}
-      KIT_MODULE_INSTALL_LIB_DIR ${VTK_VMTK_MODULE_INSTALL_LIB_DIR}
+      KIT_MODULE_INSTALL_LIB_DIR ${VTK_VMTK_INSTALL_LIB_DIR}
```

Comment:
This change prevents the vtmvtk** executables from being installed on the python path.

Reply:
Thanks for reviewing.

Since [vtkMacroKitPythonWrap.cmake](https://github.com/jcfr/vmtk/blob/2b2b250a6da6717270af96ebc6fbfa357dd4ec5c/vtkVmtk/vtkMacroKitPythonWrap.cmake) only take care of compiled and installing the wrapping of the libraries, which executable install rules are you referring to ?

Reply:
I don't know where it is defined in the code, but I am referring to how the vtkVmtk{kit_name}.so files get placed into the $VMTKHOME/Install/lib/pythonX.X/site-packages/vmtk/ directory so they can be found and read in by (vtkvmtk.py)[https://github.com/vmtk/vmtk/blob/master/vtkVmtk/vtkvmtk.py] at runtime.